pkg/app/config.go

```diff
@@ -11,12 +11,13 @@ import (
 )
 
 type Config struct {
-	Registry   apb.RegistryConfig
-	Dao        dao.Config
-	Log        LogConfig
-	Openshift  apb.ClusterConfig
-	ConfigFile string
-	DevBroker  bool
+	Registry        apb.RegistryConfig
+	Dao             dao.Config
+	Log             LogConfig
+	Openshift       apb.ClusterConfig
+	ConfigFile      string
+	DevBroker       bool
+	LaunchApbOnBind bool
```

Comment:
I would move DevBroker & LaunchApbOnBind to a BrokerConfig struct. Then make Config struct have a
BrokerConfig.

```
type Config struct {
...
BrokerConfig BrokerConfig
}
```

Then make BrokerConfig:
```
type BrokerConfig struct {
DevBroker bool
LaunchApbOnBind bool
}
```